src/test/java/io/openliberty/tools/intellij/it/SingleModMPProjectTestCommon.java

```diff
@@ -923,6 +924,69 @@ public static void prepareEnv(String projectPath, String projectName) {
                 "prepareEnv. Exit. ProjectName: " + projectName);
     }
 
+    /**
+     * Tests that when a Liberty run configuration is removed, any custom start parameters
+     * linked to the Liberty module are also cleared.
+     */
+    @Test
+    @Video
+    public void testCustomStartParametersClearedOnConfigRemoval() {
+        String testName = "testCustomStartParametersClearedOnConfigRemoval";
+        String absoluteWLPPath = Paths.get(getProjectsDirPath(), getSmMPProjectName(), getWLPInstallPath()).toString();
+
+        // Remove all existing configurations for a clean state.
+        UIBotTestUtils.deleteLibertyRunConfigurations(remoteRobot);
+
+        // Trigger the start with parameters configuration dialog.
+        UIBotTestUtils.runLibertyActionFromLTWDropDownMenu(remoteRobot, "Start...", false, 3);
+
+        // Run the configuration dialog.
+        UIBotTestUtils.runStartParamsConfigDialog(remoteRobot, getStartParamsDebugPort());
+
+        try {
+            // Validate that the project started.
+            TestUtils.validateProjectStarted(testName, getSmMpProjResURI(), getSmMpProjPort(), getSmMPProjOutput(), absoluteWLPPath, false);
+
+            // Expand the Project tree and open server.env file
+            UIBotTestUtils.openFile(remoteRobot, getSmMPProjectName(), "server.env", getServerEnvPath());
+
+            // Check if the specified Custom debug port is used.
+            TestUtils.checkCustomDebugPort(absoluteWLPPath);
+
+        } catch (IOException e) {
+            System.err.println("Error reading the file: " + e.getMessage());
+
+        } finally {
+            // Stops the Liberty server if necessary.
+            UIBotTestUtils.cleanupAndStopServerIfNeeded(remoteRobot, absoluteWLPPath, getSmMPProjectName(), testName);
+        }
+
+        // Cleanup configurations.
+        UIBotTestUtils.deleteLibertyRunConfigurations(remoteRobot);
+
+        // Start dev mode.
+        UIBotTestUtils.runLibertyActionFromLTWDropDownMenu(remoteRobot, "Start", true, 3);
```

Comment:
Is there a reason why the `usePlayButton` boolean param is set to `true` here and set to `false` in the first call (line 941)?

Reply:
We can use both true or false, that is if play button should be used to run the action or not. I have changed both to true.